src/FileProcessor.js

```diff
@@ -23,7 +23,7 @@ class FileProcessor {
     const processIndex = async (index) => {
       if (index === totalChunks || index === endIndex) {
         debug('File process complete')
-        return
+        return true
```

Comment:
Why do we need this function to return `true` or `false`? I don't see the return value actually used anywhere

Reply:
Because eslint complains that the return types are different otherwise. It probably deduces one of them as a Promise and the others as undefined. If this was TypeScript I think specifying Promise<void> as the return type would do the trick. But given that this is JS, I do not know of any other way to make eslint happy. If you know of any, I am happy to update the PR. 🙂

Reply:
Interesting, guess we can leave it as is then!